draft-ietf-quic-qpack.md

```diff
@@ -965,9 +965,8 @@ entry.
 
 ### Indexed Header Field
 
-An indexed header field representation identifies an entry in either the static
-table or the dynamic table and causes that header field to be added to the
-decoded header list.
+An indexed header field representation identifies an entry in the static table,
+or an entry in the dynamic table with an absolute index less than the Base.
```

Comment:
I now find the comparison of absolute index and "Base" (which I assert is a count) to be awkward.  Does it read better with 

> less than the *value of* the Base

Reply:
We can tackle this in another PR.  This is all goodness so I'm merging now :)